pkg/kubecost/status.go

```diff
@@ -33,3 +33,31 @@ type FileStatus struct {
 	Errors       []string          `json:"errors,omitempty"`
 	Warnings     []string          `json:"warnings,omitempty"`
 }
+
+// CloudStatus describes CloudStore metadata
+type CloudStatus struct {
+	CloudAssets    *CloudAssetStatus     `json:"cloudAssets,omitempty`
+	Reconciliation *ReconciliationStatus `json:"reconciliation,omitempty"`
+}
+
+// CloudAssetStatus describes CloudAsset metadata of a CloudStore
+type CloudAssetStatus struct {
+	Coverage    Window            `json:"coverage"`
+	LastRun     time.Time         `json:"lastRun"`
+	NextRun     time.Time         `json:"NextRun"`
```

Comment:
Probably better to lower-case the "N" here and in the other struct, too. Would you agree?